ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -172,3 +179,18 @@
     src: sudoers
     dest: /etc/sudoers.d/dev-desktop
     mode: 0440
+
+- name: Reboot to apply sysctl changes
```

Comment:
I am not sure if rebooting as part of the role can cause issues with existing workflows. A reboot is definitely required after changing `kernel.yama.ptrace_scope`, but maybe there is a better way to ensure that it happens.

(This will also come up with the quotas, since they require a restart as well to enable the feature.)

Reply:
Can we do the reboot with a handler? That way if multiple steps require a reboot we only do it once.

Reply:
There is also a [reboot module](https://docs.ansible.com/ansible/2.7/modules/reboot_module.html).

Reply:
Oh my. The module is so nice. Role has been updated to use a handler and the module. :+1: